### ISSUE TITLE
Players are able to move NPCs with ground pound

### ISSUE THREAD

**Issue by Eufranio** [Maxqia/DoubleJumpMinus#2]
You could check if the entity affected is a real player, maybe?

**Comment by Maxqia** [comment]
I'd think that would be a problem with the plugin providing the NPC ..., what plugin are you using for that?

**Comment by Eufranio** [comment]
The plugin is Citizens, and the NPCs won't be moved by gravity, hit, walking on it, explosions, anyway. The ONLY way that I found to move them until now is this plugin, so I bet this isn't a issue with Citizens.

**Comment by Maxqia** [comment]
Ehhhh, Citizens allows setVelocity on their NPCs ..., also dunno how'd I'd exclude them. You could always just remove the DJN.damage permission

**Comment by Eufranio** [comment]
The damage is already set to false, they are Moved not killed. You could check if the affected player is a NPC.

**Comment by Eufranio** [comment]
Also, even with speed 0 they are still being throwed away.

**Comment by Maxqia** [comment]
No, the permission node "DJN.damage", not the option... it should disable that part. Speed 0 only sets the height velocity BTW.
